plaso/parsers/nids_fast.py

```diff
@@ -0,0 +1,327 @@
+      r'{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((: [0-9a-fA-F]'
+      r'{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4})'
+      r'{0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2 [0-4]'
+      r'|1{0,1}[0-9]){0,1}[0-9]).){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}'
+      r'[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}'
+      r'[0-9]).){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))'
+      # Source IPv4 address
+      r'|(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?'
+      r'[0-9][0-9]?)\.(25[0-5] |2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|'
+      r'2[0-4][0-9]|[01]?[0-9][0-9]?))'
+      # Optional source port
+      r'(:\d*)?\s*'
+      # Separator '->'
+      r'\-\>\s*'
+      # Destination IPv6-address
+      r'((([0-9a-fA-F]{1,4}:){7,7}[0-9a-fA-F]{1,4}|([0-9a-fA-F]{1,4}:){1,7}:'
+      r'|([0-9a-fA-F]{1,4}:){1,6}:[0-9a-fA-F]{1,4}|([0-9a -fA-F]{1,4}:){1,5}'
+      r'(:[0-9a-fA-F]{1,4}){1,2}|([0-9a-fA-F]{1,4}:){1,4}(:[ 0-9a-fA-F]{1,4})'
+      r'{1,3}|([0-9a-fA-F]{1,4}:){1,3}(:[0-9a-fA-F]{1,4}){1,4} |([0-9a-fA-F]'
+      r'{1,4}:){1,2}(:[0-9a-fA-F]{1,4}){1,5}|[0-9a-fA-F]{1,4}:((: [0-9a-fA-F]'
+      r'{1,4}){1,6})|:((:[0-9a-fA-F]{1,4}){1,7}|:)|fe80:(:[0-9a-fA-F]{0,4})'
+      r'{0,4}%[0-9a-zA-Z]{1,}|::(ffff(:0{1,4}){0,1}:){0,1}((25[0-5]|(2 [0-4]'
+      r'|1{0,1}[0-9]){0,1}[0-9]).){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}'
+      r'[0-9])|([0-9a-fA-F]{1,4}:){1,4}:((25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}'
+      r'[0-9]).){3,3}(25[0-5]|(2[0-4]|1{0,1}[0-9]){0,1}[0-9]))'
+      # Destination IPv4 address
+      r'|(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?'
+      r'[0-9][0-9]?)\.(25[0-5] |2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|'
+      r'2[0-4][0-9]|[01]?[0-9][0-9]?))'
+      # Optional destination port
+      r'(:\d*)?$'
+  )
+  _VERIFICATION_REGEX = re.compile(_FASTLOG_VERIFICATION_PATTERN)
+
+  _PYPARSING_COMPONENTS = {
+      'year': tp.PyparsingConstants.TWO_DIGITS.setResultsName('year'),
+      'month': tp.PyparsingConstants.TWO_DIGITS.setResultsName('month'),
+      'day': tp.PyparsingConstants.TWO_DIGITS.setResultsName('day'),
+      'hour': tp.PyparsingConstants.TWO_DIGITS.setResultsName('hour'),
+      'minute': tp.PyparsingConstants.TWO_DIGITS.setResultsName('minute'),
+      'second': tp.PyparsingConstants.TWO_DIGITS.setResultsName('second'),
+      'fractional_seconds': pyparsing.Word(pyparsing.nums).setResultsName(
+          'fractional_seconds'
+      ),
+      'rule_id': pyparsing.Combine(
+          tp.PyparsingConstants.INTEGER
+          + ':'
+          + tp.PyparsingConstants.INTEGER
+          + ':'
+          + tp.PyparsingConstants.INTEGER
+      ).setResultsName('rule_id'),
+      'message': pyparsing.Combine(
+          pyparsing.OneOrMore(
+              pyparsing.Word(pyparsing.printables, exclude_chars='["')
+              | pyparsing.White(" ", max=2)
+          )
+      ).set_results_name('message'),
```

Comment:
I assume you mean setResultsName ?

Reply:
Yes, changed in fb491fa71f7db0725a9c96c0c24f8fd0805f64e0.